documentation update - steps to update from v0.7 to v0.8 (#1433)
microsoft/finops-toolkit: docs-mslearn/toolkit/hubs/upgrade.md

1. Delete FinOps hubs 0.7 networking resources:
   1. Open the FinOps hub resource group in the Azure portal.
   2. Delete all private endpoints within the resource group.
   3. Delete all private DNS zones within the resource group.
   4. Delete the virtual network.  If errors are encountered:
      - Confirm no private endpoints or DNS zones remain.
      - Check the connected devices tab and remove any lingering resources to ensure the virtual network is not in use.

> 2. Redeploy the template by using the **same inputs** from the previous deployment
>    1. Subscription
>    2. Resource group
>    3. location
>    4. hubName
>    5. dataExplorerName (if previously deployed)
>    6. Choose the correct Aure Data Explorer SKU for the deployment, ensuring it matches with the SKU which is currently in use

Reviewer: ```suggestion
2. Redeploy the template by using the **same inputs** from the previous deployment:
   - Subscription
   - Resource group
   - Location
   - Hub name
   - Data Explorer cluster name (recommended when monitoring over $100K in total spend)
   - Data Explorer SKU (if a cluster name is specified)
```

> 3. Verify that the Azure Data Factory triggers associated with Storage events have a status of 'started

Reviewer: ```suggestion
3. Verify that the Data Factory triggers are all started.
```
